event/src/unvalidated/event.rs

```diff
@@ -345,9 +345,9 @@ pub struct RawTimeEvent {
 impl Debug for RawTimeEvent {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         f.debug_struct("RawTimeEvent")
-            .field("id", &self.id.to_string())
-            .field("prev", &self.prev.to_string())
-            .field("proof", &self.proof.to_string())
+            .field("id", &format!("{:?}", &self.id))
```

Comment:
Probably nicer to use the `write!` macro with the formatter for these debug impls e.g. `write!(f, "{:?}, self.id)`

Reply:
actually, nvm.. overlooked the `debug_struct` restraint... there is `format_args!` but it's basically same same